src/Tickets/Commerce/Gateways/Stripe/Settings.php

```diff
@@ -210,6 +210,23 @@ public function get_settings() {
 				'html'            => $this->get_connection_settings_html(),
 				'validation_type' => 'html',
 			],
+		];
+
+		// If gateway isn't connected/active, only show the connection settings.
+		$is_connected = tribe( Gateway::class )->is_active();
```

Comment:
I think this would be `tribe( Merchant::class )->is_connected();` with the most recent changes, maybe @moraleida can clarify that

Reply:
`tribe( Gateway::class )->is_active()` adds the `Gateway::should_show()` check. Otherwise, I believe it's the same. @moraleida which would be better to use, in your opinion?

Reply:
That is correct. The `Client` class does not exist anymore. Its methods were migrated to either the new `Payment_Intent` class or `Merchant`